azure-governance-cloudwise/OMSAzureDashboards/Archived/OMSSQLAzureMonitoringSolution.json

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
if these are truly "archived" just delete them.  If they're still needed as part of the sample, move to the nested folder - if they're part of a standalone sample (that's not deprecated) put them in another folder.